templates/default-distributed-db-config.j2

```diff
@@ -0,0 +1,11 @@
+{
+  "autoDeploy": {{orientdb_distributed_db_config.autoDeploy | to_json }},
+  "hotAlignment": {{orientdb_distributed_db_config.hotAlignment | to_json }},
+  "executionMode": "{{orientdb_distributed_db_config.executionMode}}",
+  "readQuorum": {{orientdb_distributed_db_config.readQuorum}},
+  "writeQuorum": {{orientdb_distributed_db_config.writeQuorum}},
+  "failureAvailableNodesLessQuorum": {{orientdb_distributed_db_config.failureAvailableNodesLessQuorum | to_json}},
+  "readYourWrites": {{orientdb_distributed_db_config.readYourWrites | to_json}},
+  "servers": {{orientdb_distributed_db_config.servers | to_json}},
```

Comment:
It is present in documentation 
![screenshot from 2016-06-12 00-57-41](https://cloud.githubusercontent.com/assets/4499780/15987655/b44a0326-3038-11e6-984b-7391f8957df7.png)

you can try to run this playbook without this commit and see that it generates: default-distributed-db-config.json file